src/index.tsx

```diff
@@ -59,9 +60,11 @@ try {
   cookieRefreshTimer();
 
   ReactDOM.render(
-    <ErrorBoundary errorComponent={() => <></>} errorHandlerCallback={redirectToError}>
-      <App containerElement={applicationRoot} />
-    </ErrorBoundary>,
+    <RecoilRoot>
```

Comment:
It occurs to me that if `RecoilRoot` was moved inside the `App` component, then it might be slightly better, though I wanted the ability to potentially manage errors or the `ErrorBoundary` .. OTOH, maybe the point of the ErrorBoundary here is that it handles everything outside of the React componentry. Thoughts?